{"kind":"symbol","primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"invalidIdentifier","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"message"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":")","kind":"text"}],"platforms":["iOS"],"languages":["swift"]}]}],"metadata":{"externalID":"s:10DigiMeCore8SDKErrorO17invalidIdentifieryACSS_tcACmF","modules":[{"name":"DigiMeCore"}],"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"invalidIdentifier","kind":"identifier"},{"text":"(","kind":"text"},{"text":"message","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":")"}],"roleHeading":"Case","title":"SDKError.invalidIdentifier(message:)","symbolKind":"case"},"abstract":[{"text":"Health Data Store. Invalid identifier.","type":"text"}],"sections":[],"hierarchy":{"paths":[["doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore","doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore\/SDKError"]]},"schemaVersion":{"patch":0,"minor":3,"major":0},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore\/SDKError\/invalidIdentifier(message:)"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/digimecore\/sdkerror\/invalididentifier(message:)"]}],"references":{"doc://digimecore.DigiMeCore/documentation/DigiMeCore/SDKError":{"identifier":"doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore\/SDKError","role":"symbol","title":"SDKError","url":"\/documentation\/digimecore\/sdkerror","kind":"symbol","navigatorTitle":[{"text":"SDKError","kind":"identifier"}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"SDKError"}],"abstract":[{"type":"text","text":"SDK Errors"}],"type":"topic"},"doc://digimecore.DigiMeCore/documentation/DigiMeCore":{"role":"collection","title":"DigiMeCore","url":"\/documentation\/digimecore","abstract":[],"kind":"symbol","identifier":"doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore","type":"topic"},"doc://digimecore.DigiMeCore/documentation/DigiMeCore/SDKError/invalidIdentifier(message:)":{"abstract":[{"text":"Health Data Store. Invalid identifier.","type":"text"}],"role":"symbol","type":"topic","kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"invalidIdentifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"message"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":")","kind":"text"}],"url":"\/documentation\/digimecore\/sdkerror\/invalididentifier(message:)","identifier":"doc:\/\/digimecore.DigiMeCore\/documentation\/DigiMeCore\/SDKError\/invalidIdentifier(message:)","title":"SDKError.invalidIdentifier(message:)"}}}